kitty/mouse.c

```diff
@@ -698,6 +700,25 @@ scroll_event(double UNUSED xoffset, double yoffset, int flags) {
     }
 }
 
+static PyObject*
+send_mouse_event(PyObject *self UNUSED, PyObject *args) {
+    Screen *screen;
+    unsigned int x, y;
+    int button, action, mods;
+    if (!PyArg_ParseTuple(args, "O!IIiii", &Screen_Type, &screen, &x, &y, &button, &action, &mods)) return NULL;
+
+    MouseTrackingMode mode = screen->modes.mouse_tracking_mode;
+    if (mode == ANY_MODE || (mode == MOTION_MODE && action != MOVE) || (mode == BUTTON_MODE && (action == PRESS || action == RELEASE))) {
```

Comment:
Shouldnt this be action == move??

Reply:
No, when the mode is `MOTION_MODE`, the terminal should send `PRESS`, `RELEASE` and `DRAG` actions, but not `MOVE` actions.

Reply:
I could write `mode == MOTION_MODE && (action == PRESS || action == RELEASE || action == DRAG)` if that's clearer?

It would be nice with some tests for this function, but not sure how to test the whole function? I guess the easiest would be to pull out this if to a separate function and test that?